python/ray/cloudpickle/cloudpickle.py

```diff
@@ -572,8 +572,8 @@ def extract_code_globals(cls, co):
                 # PyPy "builtin-code" object
                 out_names = set()
             else:
-                out_names = set(names[oparg]
-                                for op, oparg in _walk_global_ops(co))
```

Comment:
Let's not make any changes to this file as it is directly copied from the cloudpickle repository.

Reply:
Good point. Put in a PR w/cloudpickle to change that though.